x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.test.ts

```diff
@@ -504,7 +504,7 @@ describe('rules_notification_alert_type', () => {
         await alert.executor(payload);
         expect(logger.error).toHaveBeenCalled();
         expect(logger.error.mock.calls[0][0]).toContain(
-          'An error occurred during rule execution: message: "Threat Match rule is missing threatQuery and/or threatIndex and/or threatMapping: threatQuery: "undefined" threatIndex: "undefined" threatMapping: "undefined"" name: "Detect Root/Admin Users" id: "04128c15-0d1b-4716-a4c5-46997ac7f3bd" rule id: "rule-1" signals index: ".siem-signals"'
+          'An error occurred during rule execution: message: "Indicator match is missing threatQuery and/or threatIndex and/or threatMapping: threatQuery: "undefined" threatIndex: "undefined" threatMapping: "undefined"" name: "Detect Root/Admin Users" id: "04128c15-0d1b-4716-a4c5-46997ac7f3bd" rule id: "rule-1" signals index: ".siem-signals"'
```

Comment:
Should the wording here match the UI when it comes to `threatIndex`, `threatMapping`, `threatQuery` or at least let the user know that's what they correspond to? Or I guess that's particular to the field name right? Would this be an error the user sees in the front end ever?

Reply:
I take back my comment, that would add a lot of overhead any time we decide to switch what we call it in the UI.

Reply:
I view log files as something that is for both the users and us to look through so when UI wording changes I might not be easily able to keep files and REST up to date (or REST would break) but for things like log file messages I try to update them to match the UI and the wording in people's heads.

I try to take extra time to make the logs more human readable for users more so than even myself even though the majority of the time they end up in the forums. I kind of would like to make them more clear.

We don't i18n them though so at the moment they're not as accessible as we would like them to be.

Reply:
Yea, I was just thinking about whether they'd get confused with the `threat` vs `indicator` lingo in the error message. i18n eventually would be super nice too.